src/components/UniversalNavbarExpanded/DropdownNav/DropdownChildren.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Layout, Footnote } from '../../index'
+
+// Parent component (UniversalNavbar) siblings
+import NavLink from '../NavLink'
+
+// DropdownChildren sibling components
+import DropdownCta from './DropdownCta'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownChildren.module.scss'
+
+/**
+ * Content (links) displayed in the dropdown menu after hovering the parent element
+ *
+ * @param {string} extraClass - Extra top level class
+ * @param {object} child - CTA copy, url and link items with titles/urls
+ * @param {object} LinkComponent - Agnotistic Reach and React Router Link (ex. Gatsby's <Link>)
+ *
+ * @return {JSX.Element}
+ */
+const DropdownChildren = ({ extraClass, child, LinkComponent }) => {
+  // Split the links for the subnav into two columns
+  const columns = [[], []]
+  const childItems = get(child, 'subnav.items')
+
+  for (var i = 0; i < childItems.length; i++) {
```

Comment:
I think we could refactor to use es6 map, but definitely at minimum should replace `var` with `let`

Reply:
Will do.

Reply:
Agree with Rob, it may be better to just use two `.filter` statements based on the `i % 2 === 0` criteria

Reply:
Not sure what you were thinking with `.filter` in combination with `.map` or if that's even what you meant, but I implemented what we touched on IRL, using the modulus condition as the index.
```
childItems.map((childItem, i) => {
    columns[(i % 2 === 0 ? 0 : 1)].push(childItems[i])
  })
```

Reply:
8b0d3cf